src/NzbDrone.Core/DecisionEngine/Specifications/AcceptableSizeSpecification.cs

```diff
@@ -59,15 +59,15 @@ public Decision IsSatisfiedBy(RemoteAlbum subject, SearchCriteriaBase searchCrit
             }
             else
             {
-                var maxSize = qualityDefinition.MaxSize.Value.Megabytes();
+                var maxSize = qualityDefinition.MaxSize.Value.Kilobits();
 
                 //Multiply maxSize by Album.Duration
                 maxSize = maxSize * albumsDuration;
 
                 //If the parsed size is greater than maxSize we don't want it
                 if (subject.Release.Size > maxSize)
                 {
-                    var runtimeMessage = $"{albumsDuration}min";
+                    var runtimeMessage = $"{albumsDuration}sec";
 
                     _logger.Debug("Item: {0}, Size: {1} is greater than maximum allowed size ({2}), rejecting.", subject, subject.Release.Size, maxSize);
```

Comment:
Lets add the runtime message back into the debug log, seems it got removed sometime int he past. 

it was `_logger.Debug("Item: {0}, Size: {1} is greater than maximum allowed size ({2} bytes for {3}), rejecting.", subject, subject.Release.Size, maxSize, runtimeMessage);`